backend/app/api/platform/models/tasks.py

```diff
@@ -11,3 +11,14 @@ class AddEventRequest(BaseModel):
 
 class RemoveEventRequest(BaseModel):
     event_name: str
+
+
+class FetchSpansRequest(BaseModel):
+    task_id: str
+    project_id: str
+
+
+class TaskSpans(BaseModel):
+    task_id: str
+    project_id: str
+    spans: List[Dict[str, object]]
```

Comment:
Can we develop this structure because all spans should have a common basic structure ?

Reply:
Added some more details about what we know